dot/sync/chain_processor.go

```diff
@@ -238,6 +238,7 @@ func (s *chainProcessor) handleBlock(block *types.Block) error {
 	}
 
 	logger.Debugf("🔗 imported block number %d with hash %s", block.Header.Number, block.Header.Hash())
+	// should we announce the block here?
```

Comment:
did you figure it out? Or asking reviewers? :thinking:

Reply:
Yeah, I figured out we should do the block announcement right after it is imported